lib/dvc_variable.dart

```diff
@@ -1,10 +1,15 @@
+import 'package:logger/logger.dart';
+import 'package:collection/collection.dart';
+
 enum VariableType {
   string,
   boolean,
   number,
   json
 }
 
+final _logger = Logger();
```

Comment:
are we using the logger here?

Reply:
No, I was going to use it originally to log an error in the optional `orElse` when using `firstWhere` but I wasn't able to get it to work. Will remove!